### ISSUE TITLE
Can not get WebView2 app to deploy

### ISSUE THREAD

**Issue by WayneBBB** [MicrosoftEdge/WebView2Feedback#1259]
I'm running into a little issue which I'm not sure how to solve.    I'm trying to deploy my win32 WebView2 app, and I am using a SetupProject generated with the Microsoft's Visual Studio Installer Projects extension.  I have Microsoft Edge WebView2 Runtime, version 90.0.818.51 installed on my machine.  Everything works correctly when run by the debugger in Visual Studio.

In the past when I did a trial deployment, the target deployment folder would get the WebView2Loader.dll, and a associated folder called "MyProjectName".exe.WebView2, which appeared to be generated automatically as part of the deployment; and everything worked as expected.  Now, when I try to deploy to my deployment folder on my development machine, the target folder gets the WebView2Loader.dll, but no associated folders are generated.  When I run the app, it is definitely not hooking to WebView2 RunTime (everything else seems to be working.).  

I have tried loading the "MyProjectName".exe.WebView2 folder from the visual studio folder to the deployment target folder, but that didn't work. I have tried unloading/reloading WebView2 RunTime with my app absent, and then tried reloading my app.  But that didn't work.  What can I do to fix this?

**Comment by WayneBBB** [comment]
champnic:  Thanks for the response and it provided me some insights I need.   I found the problem.  There is something seriously wrong with Microsoft.Web.WebView2 release 1.0.818.41, at least for Win32.  I reverted to release 1.0.774.44 and everything runs correctly and I am using Evergreen.   What I was finding with release 1.0.818.41 was that the app didn't find  WebView2.h.  So I pointed the compiler to WebView2.h;  rebuilt;  WebView2.h was found, but then the Linker complained that it couldn't open WebView2Loader.dll.lib.  So I pointed the Linker to WebView2Loader.dll.lib and rebuilt, and then the complier complained about a runtime error.  I could never get to the point of a .exe file being built.   I found this problem by trying to go back to basics and build the getting Started app.

As an aside, I would like to let you know that I have been very pleased with my WebView2 win32 experience.  I am a newbie to c++ and win32.  So far I have had a good learning experience and have made great progress.
